### PR TITLE
Increased color contrast when buttons overlap selected rows

### PR DESCRIPTION
**Pull request by jonbell-lot23**
We have a long-standing issue where our blue time travel button disappears when it overlaps on a selected row. I've been playing around with ways to make this work better. Not just here, but how the colors can be applied across the system.

Current approach on left, proposed on right:
image
This is a no-brainer.

image
And this is a nice standardization/cleanup in the library.

### CODE REVIEW

**Comment by jasonLaster** [comment]
I like this better. Also winter what it would look like with the lighter blue row background and the primary blue button.

**Comment by jonbell-lot23** [comment]
Following up on discord comment from yesterday: yeah, I like the green too but I also think we can stick with blue for now and keep the green in our back pocket. It's coming (probably)! Doesn't need to be rushed out now though.

**Comment by bvaughn** [comment]
I like this change but it only update some parts of our app, so it causes inconsistencies. For example the Network panel looks like this now
Screen Shot 2023-11-09 at 4 46 13 PM

but the Elements and React panels still look like this
Screen Shot 2023-11-09 at 4 47 20 PM
Screen Shot 2023-11-09 at 4 47 25 PM

File: packages/replay-next/pages/variables.css [hunk 1]
```diff
@@ -709,7 +716,7 @@
   --theme-text-field-bgcolor: #f6f6f6;
   --theme-text-field-color: var(--body-color);
   --theme-text-field-placeholder-color: #a9a9b1;
-  --team-row-active: #3f434a;
+  --team-row-active: #163e58; /* the dark version of --primary-accent-dimmed */
```
Comment (bvaughn) [inline packages/replay-next/pages/variables.css:719]: btw
```suggestion
  --team-row-active: var(--primary-accent-dimmed);
```

**Comment by bvaughn** [comment]
Noticed light theme has a contrast issue with the duration bubble
Screen Shot 2023-11-09 at 4 50 33 PM

File: packages/replay-next/pages/variables.css (hunk 1 above)
Comment (jonbell-lot23) [reply packages/replay-next/pages/variables.css:719]: Usually variables are great. But in this case it's the left-hand column in the library, which always stays dark. Using the variable here means it uses the wrong color when in light theme.

**Comment by jonbell-lot23** [comment]
Oh good -- if you think it works fine in those cases, I'll add them here. I was going to address that in a follow-up because I assumed the HTML colors would cause more issues and edge cases.

(Philosophically there are two ways to look at this -- find everything related and put it into a single PR versus stage things out, but we don't have to get into that here, happy to put this into one)

File: packages/replay-next/pages/variables.css (hunk 1 above)
Comment (bvaughn) [reply packages/replay-next/pages/variables.css:719]: Oh I see. Hmm 🤔 That's pretty easy to get mixed up.

Comment (jonbell-lot23) [reply packages/replay-next/pages/variables.css:719]: Indeed. I run against this every. single. time.

**Comment by jonbell-lot23** [comment]
@bvaughn I am having trouble figuring out what file changes the selected color in the react devtools. I tried deleting everything from variables.css but the blue still comes through, meaning it's not using variables. Then I looked in files like src/ui/components/SecondaryToolbox/react-devtools/components/ReactDevToolsListItem.module.css but I couldn't manage to find the right spot.

**Comment by jonbell-lot23** [comment]
> I like this change but it only update some parts of our app, so it causes inconsistencies. 

Updated these:

image

image

> Noticed light theme has a contrast issue with the duration bubble.

Updated these:

image

image

**Review by ryanjduffy — APPROVED** [review]
:+1: Adding some snapshot tests for these changes would be nice too

**Pull request merged**